docs/src/components/ReanimatedExamples/FadeTileExample/index.jsx

```diff
@@ -41,26 +41,27 @@ function mergeStyles(baseStyles, additionalStyles) {
   });
 }
 
+let nextLetter = 'A';
```

Comment:
why it is not a react state?

Reply:
We wanted to avoid triggerring unncessary re-renders